### DISCUSSION TITLE
[Script request] PHPIPAM

### DISCUSSION THREAD

**Discussion by darthrater78** [community-scripts/ProxmoxVE#594] [Request script]
### Application Name

PHPIPAM

### Website

https://phpipam.net/

### Description

Netbox is already a script, a great other choice is phpipam. It's easy to use and can do scheduled scans out of the box to automatically update inventory.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bvdberg01** [comment]
I can look into this one.

**Reply by MickLesk** [reply]
Slow down. We have at the moment to many PRs Open. This Script is currently Not in Pipeline.

**Reply by bvdberg01** [reply]
Ah sorry, didn't see that it was not in pipeline. But will it be added to the pipepline? I had the script almost done 😅

**Reply by MickLesk** [reply]
Please wait 1 Week. I Prepare a bigger change. After this, yes

**Reply by bvdberg01** [reply]
> Please wait 1 Week. I Prepare a bigger change. After this, yes

Is it already ok to open pr for phpipam now? Or still need to wait?

**Reply by MickLesk** [reply]
is this in ready for pr? ^^

**Reply by raspberrypisig** [reply]
Any progress?

**Reply by MickLesk** [reply]
Not yet in the script pipeline.

**Reply by darthrater78** [reply]
Any ETA?

**Reply by michelroegl-brunner** [reply]
Not at the moment. We are working on many background tasks atm.

**Reply by darthrater78** [reply]
I just discovered the project board, wow you aren't kidding. Thanks for the response!

**Reply by MickLesk** [reply]
I hope that we will be through with the majority by 01.03, after that I will evaluate incoming requests until 15.02 for the next preliminary run. Unless someone wants to do it and creates the script before then

**Reply by bvdberg01** [reply]
I had the script almost done, only needs to be updated to the latest standards.

**Comment by KyruJames** [comment]
id love this too but im going to install it myself

**Reply by tremor021** [reply]
https://community-scripts.github.io/ProxmoxVE/scripts?id=phpipam

Did you even looked at the website? :)

**Comment by tremor021** [comment]
Done in #1503
